graylog2-server/src/main/java/org/graylog2/shared/rest/resources/system/RemoteDeflectorResource.java

```diff
@@ -20,6 +20,6 @@
 import retrofit2.http.POST;
 
 public interface RemoteDeflectorResource {
-    @POST("/system/deflector/cycle")
+    @POST("system/deflector/cycle")
```

Comment:
Ok bummer. I was asking because at least the audit log plugin needs the corresponding change in this case.